tests/fxl-page-spread_left/OPS/page_001.xhtml

```diff
@@ -0,0 +1,10 @@
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:epub="http://www.idpf.org/2007/ops" xml:lang="en">
+  <head>
+    <title>rendition:page-spread- left</title>
+    <meta name="viewport" content="width=900, height=600" />
+    <link rel="stylesheet" href="fixed.css" />
+  </head>
+  <body id="p1">
+  	<p>Page 1. This test passes if the next page is displayed on the left hand side of the synthetic spread.</p>
```

Comment:
Ultra-nit: the following pages all say "left side" instead of "left hand side". Cut "hand" to match.